recipes/bamtools/meta.yaml

```diff
@@ -1,34 +1,34 @@
-{% set version = "2.4.1" %}
-
+{% set version = "2.5.1" %}
+{% set sha256 = "4abd76cbe1ca89d51abc26bf43a92359e5677f34a8258b901a01f38c897873fc" %}
 package:
   name: bamtools
-  version: {{ version }}
+  version: '{{version}}'
 
 source:
-  fn: v{{ version }}.tar.gz
-  url: https://github.com/pezmaster31/bamtools/archive/v{{ version  }}.tar.gz
-  sha256: 933a0c1a83c88c1dac8078c0c0e82f6794c75cb927265399404bc2cc2611204b
+  fn: bamtools-{{version}}.tar.gz
+  url: https://github.com/pezmaster31/bamtools/archive/v{{version}}.tar.gz
+  sha256: '{{sha256}}'
   patches:
     # Install bamtools in lib instead of bamtools lib, thanks to homebrew-science:
     # https://github.com/Homebrew/homebrew-science/blob/master/bamtools.rb
     - bamtools_lib.diff
```

Comment:
Remove this and override GNUInstallDirs properly: `-DCMAKE_INSTALL_LIBDIR=${PREFIX}/lib`. The patch is the reason why the CI is failing. I have fixed bamtools properly.